pyproject.toml

```diff
@@ -1,12 +1,101 @@
 [build-system]
-requires = [
-    "setuptools>=42",
-    "wheel"
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "oteapi-dlite"
+authors = [
+    {name = "SINTEF", email = "Team4.0@SINTEF.no"},
+]
+description = "DLite plugin for Open Translation Environment (OTE) API."
+readme = "README.md"
+license = {file = "LICENSE"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
Python 3.11 can be added, since we are already testing for it. 
We should also add Python 3.12, but that can be in a new PR.

Reply:
3.12 fails, so not yet.